pgtap/breadthFirstSearch/binaryBreadthFirstSearch/binaryBreadthFirstSearch-rows-check.sql

```diff
@@ -0,0 +1,106 @@
+\i setup.sql
+
+SELECT plan(10);
+
+
+-- Check whether the same set of rows is always returned
+
+-- Creating the table used for the queries
+
+DROP TABLE IF EXISTS roadworks CASCADE;
```

Comment:
Why do you need roadworks?
you are creating, inserting and not using that table in any of your tests.
please delete

Reply:
Oh sorry. I was supposed to use `roadworks` here instead of `edge_table` (based on the function's documentation)

Reply:
Wrong, you have to use the edge_table

Reply:
I changed a lot of files in those 4-5 days (mainly copy-paste). That's why this error :(

Reply:
The user's documentation will be fixed, they have to use the edge_table